File: FC.App/Scripts/Shared/Models/NewsFilter.js
var FC;
(function (FC) {
    var Shared;
    (function (Shared) {
        var Models;
        (function (Models) {
            var NewsFilter = (function () {
                function NewsFilter() {
                    this.GenreIDs = new Array();
                    this.CountryIDs = new Array();
                }
                return NewsFilter;
            }());
            Models.NewsFilter = NewsFilter;
        })(Models = Shared.Models || (Shared.Models = {}));
    })(Shared = FC.Shared || (FC.Shared = {}));
})(FC || (FC = {}));

File: FC.Shared/ServerMessages/NewsFilter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FC.Shared.ServerMessages
{
    public class NewsFilter
    {
        public List<Guid?> GenreIDs { get; set; }
        public List<Guid?> CountryIDs { get; set; }
    }
}
